modular_skyrat/master_files/code/datums/quirks/negative.dm

```diff
@@ -21,3 +21,75 @@
 	qdel(staple)
+
+// Re-labels TG brainproblems to be more generic. There never was a tumor anyways!
+/datum/quirk/item_quirk/brainproblems
+	name = "Brain Degeneration"
+	desc = "You have a lethal condition in your brain that is slowly destroying it. Better bring some mannitol!"
+	medical_record_text = "Patient has a lethal condition in their brain that is slowly causing brain death."
+
+// Override of Brain Tumor quirk for robotic/synthetic species with posibrains.
+// Does not appear in TGUI or the character preferences window.
+/datum/quirk/item_quirk/brainproblems/synth
+	name = "Positronic Cascade Anomaly"
+	desc = "Your positronic brain is slowly corrupting itself due to a cascading anomaly. Better bring some liquid solder!"
+	gain_text = "<span class='danger'>You feel glitchy.</span>"
+	lose_text = "<span class='notice'>You no longer feel glitchy.</span>"
+	medical_record_text = "Patient has a cascading anomaly in their brain that is slowly causing brain death."
+	icon = "bp_synth_brain"
+	mail_goodies = list(/obj/item/storage/pill_bottle/liquid_solder/braintumor)
+	hidden_quirk = TRUE
+
+// If brainproblems is added to a synth, this detours to the brainproblems/synth quirk.
+// TODO: Add more brain-specific detours when PR #16105 is merged
+/datum/quirk/item_quirk/brainproblems/add_to_holder(mob/living/new_holder, quirk_transfer)
+	if(!(is_species(new_holder, /datum/species/robotic) && (src.type == /datum/quirk/item_quirk/brainproblems)))
+		// Defer to TG brainproblems if the character isn't robotic.
+		return ..()
+	qdel(src)
+	// TODO: Check brain type and detour to appropriate brainproblems quirk
+	var/datum/quirk/item_quirk/brainproblems/synth/_bp_synth = new
+	return _bp_synth.add_to_holder(new_holder, quirk_transfer)
+
+// Synthetics get liquid_solder with Brain Tumor instead of mannitol.
+/datum/quirk/item_quirk/brainproblems/synth/add_unique()
+	give_item_to_holder(
+		/obj/item/storage/pill_bottle/liquid_solder/braintumor,
+		list(
+			LOCATION_LPOCKET = ITEM_SLOT_LPOCKET,
+			LOCATION_RPOCKET = ITEM_SLOT_RPOCKET,
+			LOCATION_BACKPACK = ITEM_SLOT_BACKPACK,
+			LOCATION_HANDS = ITEM_SLOT_HANDS,
+		),
+		flavour_text = "These will keep you alive until you can secure a supply of medication. Don't rely on them too much!",
+	)
+
+// Override of Blood Deficiency quirk for robotic/synthetic species.
+// Does not appear in TGUI or the character preferences window.
+/datum/quirk/blooddeficiency/synth
+	name = "Hydraulic Leak"
+	desc = "Your body's hydraulic fluids are leaking through their seals."
+	medical_record_text = "Patient requires regular treatment for hydraulic fluid loss."
+	icon = "bd_synth_tint"
+	hidden_quirk = TRUE
+
+// If blooddeficiency is added to a synth, this detours to the blooddeficiency/synth quirk.
+/datum/quirk/blooddeficiency/add_to_holder(mob/living/new_holder, quirk_transfer)
```

Comment:
same goes for here

Reply:
Fixed in commit b4d597862c5e1e7aee0daec7868bdf1f6d080817